### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by stephendonner**
OK, tested locally against both prod and staging, *with* pytest-reruns and pytest-xdist==1.8 /before/ the usual pip install -Ur requirements.txt.

Also passed same environments on staging's ad-hoc jobs.

Prod: http://webqa-ci-staging1.qa.scl3.mozilla.com:8080/job/mozillians.adhoc/75/
Staging: http://webqa-ci-staging1.qa.scl3.mozilla.com:8080/job/mozillians.adhoc/76/

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
-pytest==2.7.3
+BeautifulSoup==3.2.1
+browserid
```
Comment (davehunt) [inline requirements.txt]: This was renamed to `bidpom`. I'm surprised we didn't update in all our requirements files. This can be a separate commit, or even a separate pull.

Comment (davehunt) [inline requirements.txt]: On second thoughts, let's do this in a separate patch as it might not just be a package name change - imports will also need updating.

**Comment by davehunt** [comment]
r+

**Comment by rbillings** [comment]
Tests pass locally, changes look good. R+
